examples/ssr/package.json

```diff
@@ -7,6 +7,9 @@
     "start": "razzle start",
     "start:prod": "NODE_ENV=production node build/server.js"
   },
+  "resolutions": {
+    "mini-css-extract-plugin": "^1.6.2"
```

Comment:
Can't you just upgrade `mini-css-extract-plugin` to `^1.6.2` and change the entry in `devDependencies`?

Reply:
@MonicaOlejniczak yeahhh, I thought about it, but wanted to keep it consistent with what recommended by Razzle
https://razzlejs.org/docs/upgrade-guide

Reply:
Yeah ok, looks like they use 0.9.0 for their peer deps... pretty ick

Reply:
yeahhhh, they introduced a bunch of [peerDependencies](https://github.com/jaredpalmer/razzle/issues/1677) in v4.x